specificatie/informatie-service/openapi.yaml

```diff
@@ -433,10 +437,13 @@ components:
       type: boolean
       description: |
         Gegevens mogen niet worden verstrekt aan derden / maatschappelijke instellingen.
-    InOnderzoek:
```

Comment:
waarom is dit verwijderd. InOnderzoek moet nog steeds worden gecommuniceerd als één of meerdere velden van verblijfplaats in onderzoek staan